packages/server/src/arpa_reporter/lib/audit-report.js

```diff
@@ -125,22 +132,29 @@ async function getProjectSummaryGroupedByProjectRow(data) {
             `${reportingPeriodEndDate} Total Expenditures for Awards Greater or Equal to $50k`,
             `${reportingPeriodEndDate} Total Aggregate Obligations`,
             `${reportingPeriodEndDate} Total Obligations for Awards Greater or Equal to $50k`,
-        ].forEach((columnName) => { row[columnName] = 0; });
+        ].forEach((columnName) => { projectSummaryV2Row[columnName] = 0; });
     });
 
-    row['Capital Expenditure Amount'] = 0;
+    projectSummaryV2Row['Capital Expenditure Amount'] = 0;
 
     // set values in each column
     records.forEach(async (r) => {
+        // for project summaries v2 report
         const reportingPeriodEndDate = reportingPeriods.filter((reportingPeriod) => r.upload.reporting_period_id === reportingPeriod.id)[0].end_date;
-        row[`${reportingPeriodEndDate} Total Aggregate Expenditures`] += (r.content.Total_Expenditures__c || 0);
-        row[`${reportingPeriodEndDate} Total Aggregate Obligations`] += (r.content.Total_Obligations__c || 0);
-        row[`${reportingPeriodEndDate} Total Obligations for Awards Greater or Equal to $50k`] += (r.content.Award_Amount__c || 0);
-        row[`${reportingPeriodEndDate} Total Expenditures for Awards Greater or Equal to $50k`] += (r.content.Expenditure_Amount__c || 0);
-        row['Capital Expenditure Amount'] += (r.content.Total_Cost_Capital_Expenditure__c || 0);
+        projectSummaryV2Row[`${reportingPeriodEndDate} Total Aggregate Expenditures`] += (r.content.Total_Expenditures__c || 0);
+        projectSummaryV2Row[`${reportingPeriodEndDate} Total Aggregate Obligations`] += (r.content.Total_Obligations__c || 0);
+        projectSummaryV2Row[`${reportingPeriodEndDate} Total Obligations for Awards Greater or Equal to $50k`] += (r.content.Award_Amount__c || 0);
+        projectSummaryV2Row[`${reportingPeriodEndDate} Total Expenditures for Awards Greater or Equal to $50k`] += (r.content.Expenditure_Amount__c || 0);
+        projectSummaryV2Row['Capital Expenditure Amount'] += (r.content.Total_Cost_Capital_Expenditure__c || 0);
+
+        // for kpi report
+        const currentPeriodExpenditure = r.content.Current_Period_Expenditures__c || 0;
+        kpiRow['Number of Subawards'] += (r.type === 'awards50k');
+        kpiRow['Number of Expenditures'] += (currentPeriodExpenditure > 0);
```

Comment:
```suggestion
        kpiRow['Number of Expenditures'] += Math.max(currentPeriodExpenditure, 0);
```

Reply:
I don't think currentPeriodExpenditure will be either 1 or 0. This column sound like we want to add up the unique number of currentPeriodExpenditures unless we are adding up the value for that column? 

@as1729 or @jcomeau1 can you chime in on this?

Reply:
Ohh, thank you for explaining this.  I definitely misunderstood the intent.  Again you're coercing true or false to an integer.

My change is definitely not correct then.  Here also, I would slightly prefer a ternary expression instead of a coercion, but I do not feel that strongly.